src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -42,38 +41,16 @@ public void systemLoadsManageConditions() {
 		assertNotNull(getElement(addNewCondition));
 	}
 
-	@And("User clicks on Return")
-	public void clickReturn() {
-		conditionsPage.clickReturn();
```

Comment:
I think we need to cover Return, you will need to cross check with the system to define when would the user need it so that it's covered